### PR TITLE
Add support for enabling blob cache

### PR DESCRIPTION
**Pull request by exabytes18**
This PR adds support for enabling blob caching.

### CODE REVIEW

File: src/db_options.rs [hunk 1]
```diff
@@ -3329,6 +3331,14 @@ impl Options {
         }
     }
 
+    /// Sets the blob cache.
```
Comment (zaidoon1) [inline src/db_options.rs:3334]: would be good to copy the information here https://github.com/facebook/rocksdb/blob/v9.3.1/include/rocksdb/advanced_options.h#L979-L987 to give more context on what this is.

**Comment by zaidoon1** [comment]
Based on https://github.com/facebook/rocksdb/wiki/BlobDB#column-family-options it looks like there is multiple options that we should expose as well esepcially:
```
enable_blob_files: set it to true to enable key-value separation.
```

My guess is this pr as is won't actually do anything/enable blobdb which is not what you want?

**Comment by exabytes18** [comment]
> Based on https://github.com/facebook/rocksdb/wiki/BlobDB#column-family-options it looks like there is multiple options that we should expose as well esepcially:
> 
> ```
> enable_blob_files: set it to true to enable key-value separation.
> ```
> 
> My guess is this pr as is won't actually do anything/enable blobdb which is not what you want?

Hi @zaidoon1. This PR is for enabling blobdb caching, not enabling blobdb itself (the `set_enable_blob_files()` option is already available in rust-rocksdb; please see the included test for how to enable blobdb + blob caching).

File: src/db_options.rs (hunk 1 above)
Comment (exabytes18) [reply src/db_options.rs:3334]: Ok, I've copied the text from rocksdb.

**Comment by zaidoon1** [comment]
> > Based on https://github.com/facebook/rocksdb/wiki/BlobDB#column-family-options it looks like there is multiple options that we should expose as well esepcially:
> > ```
> > enable_blob_files: set it to true to enable key-value separation.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > My guess is this pr as is won't actually do anything/enable blobdb which is not what you want?
> 
> Hi @zaidoon1. This PR is for enabling blobdb caching, not enabling blobdb itself (the `set_enable_blob_files()` option is already available in rust-rocksdb; please see the included test for how to enable blobdb + blob caching).

oh i see, this is for exposing more options not adding support for blobdb.

**Review by zaidoon1 — APPROVED** [review]

**Pull request merged**